examples/video_player/README.md

```diff
@@ -2,4 +2,4 @@
 
 This example demonstrates how to use `gtk::Video` to play videos.
 
-![Screenshot](app.png)
\ No newline at end of file
+![Screenshot](screenshot.png)
```

Comment:
the file seems to have a double extension

Reply:
all of them actually

Reply:
Ah, will fix it

Reply:
Fixed